sdk/storage/azure-storage-blob/azure/storage/blob/_blob_service_client.py

```diff
@@ -518,17 +501,16 @@ def find_blobs_by_tags(self, filter_expression, **kwargs):
             timeout=timeout,
             **kwargs)
         return ItemPaged(
-            command, results_per_page=results_per_page,
+            command, results_per_page=results_per_page, container=None,
```

Comment:
I needed to add this since we changed `container` to be required, and this is non-breaking as even currently the `FilteredBlobPaged` returned when utilizing this API has container=None
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/97888cbc-ec8f-4e1c-b2b5-f2bb6b8e6a1f)

Reply:
Ouch, will move to revert then since it's now inaccurate to claim it has non-optional 😢